code/modules/client/preference/preferences_toggles.dm

```diff
@@ -36,6 +36,16 @@
 	to_chat(usr, "You will [(prefs.toggles & CHAT_RADIO) ? "now" : "no longer"] see radio chatter from radios or speakers")
 	feedback_add_details("admin_verb","THR") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/verb/toggle_ai_voice_annoucements()
+	set name = "Hear/Silence AI Voice Announcements"
+	set category = "Preferences"
+	set desc = "Toggle hearing AI annoucements in voice form or in text form"
+	if(!holder)	return
```

Comment:
`holder` is a check for admin status (iirc). I would check if this works when you’re not an admin.

Reply:
holder is anyone with a rank including mentors

Reply:
Yeah, I'll remove that bit